docs/src/site/twiki/UserExtensions.twiki

```diff
@@ -0,0 +1,232 @@
+---+ Falcon User Extensions
+
+---++ Overview
+
+A Falcon User extension is an Extension Developer's implementation of Falcon Extension builder. A user extension can be used by all falcon users.
```

Comment:
This doesn't bring out the intent. May be something like this:
A falcon user extension is a solution template that comprises of Falcon entities that a Falcon user can build. The extension so built can be made available to and discovered by other users (sharing the same Falcon installation). Extension users can instantiate the solution template and deploy it by providing relevant parameters.